<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1517</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Configurtador</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="my_qlabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1280</width>
      <height>720</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSlider" name="R_Color">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>800</y>
      <width>22</width>
      <height>160</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="sliderPosition">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QSlider" name="G_Color">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>800</y>
      <width>22</width>
      <height>160</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="sliderPosition">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QSlider" name="B_Color">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>800</y>
      <width>22</width>
      <height>160</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="sliderPosition">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QDial" name="speed">
    <property name="geometry">
     <rect>
      <x>1390</x>
      <y>620</y>
      <width>61</width>
      <height>81</height>
     </rect>
    </property>
    <property name="minimum">
     <number>10</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
   </widget>
   <widget class="QComboBox" name="effectSelector">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>880</y>
      <width>171</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>SELECT EFFECT</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>TRAIN_BACK</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>TRAIN_FWRD</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>FLASH</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>SMOTH_BLINK</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>NONE</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>770</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string> R       G        B     A</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>1400</x>
      <y>610</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Velocidade</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>860</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Efeito do grupo</string>
    </property>
   </widget>
   <widget class="QPushButton" name="RecordButton">
    <property name="geometry">
     <rect>
      <x>1310</x>
      <y>790</y>
      <width>161</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Gravar</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="screen_line">
    <property name="geometry">
     <rect>
      <x>1310</x>
      <y>750</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>59</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>1390</x>
      <y>750</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Número da tela</string>
    </property>
   </widget>
   <widget class="LedIndicator" name="LED" native="true">
    <property name="geometry">
     <rect>
      <x>1160</x>
      <y>790</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QSlider" name="A_slider">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>800</y>
      <width>22</width>
      <height>160</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="sliderPosition">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="playButton">
    <property name="geometry">
     <rect>
      <x>1310</x>
      <y>720</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Play</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>820</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Número do Grupo</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="group">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>820</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>60</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>750</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Cor Predominante</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="selectLED">
    <property name="geometry">
     <rect>
      <x>1150</x>
      <y>850</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>LED</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="selectGroup">
    <property name="geometry">
     <rect>
      <x>1150</x>
      <y>870</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Grupo</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>1320</x>
      <y>10</y>
      <width>161</width>
      <height>541</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>1370</x>
      <y>560</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="led_number">
    <property name="geometry">
     <rect>
      <x>1210</x>
      <y>850</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>116</number>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>LedIndicator</class>
   <extends>QWidget</extends>
   <header>ledindicator.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>my_qlabel</class>
   <extends>QLabel</extends>
   <header location="global">my_qlabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
